### PR TITLE
Release v0.20.0

### PR DESCRIPTION
**Pull request by thanethomson**
[:book: CHANGELOG rendered](https://github.com/informalsystems/tendermint-rs/blob/release/v0.20.0/CHANGELOG.md#v0200)

* [ ] Referenced an issue explaining the need for the change
* [ ] Updated all relevant documentation in docs
* [ ] Updated all code comments where relevant
* [ ] Wrote tests
* [ ] Added entry in `.changelog/`

### CODE REVIEW

File: .changelog/v0.20.0/summary.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+*Jun 22, 2021*
+
+This release's number is bumped up to v0.20.0 due to two minor breaking changes
+in our public APIs (see the breaking changes section below for details).
+
+Also, since nobody was really making use of the Light Node, we decided to remove
+its crate from the repo for now. If anyone needs it back, please contact us and
+we'll restore it (although, we are considering migrating any and all binaries to
+their own repositories in future to separate framework-level concerns from
```
Comment (xla) [inline .changelog/v0.20.0/summary.md:9]:
```suggestion
their own repositories in the future to separate library-level concerns from
```

File: .changelog/v0.20.0/summary.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+operational ones).
+
+The `tendermint-p2p` crate is still undergoing significant expansion (thanks to
+@xla and @melekes!). A lot's been done and we're in the process of finalizing
```
Comment (xla) [inline .changelog/v0.20.0/summary.md:13]:
```suggestion
@xla and @melekes)! A lot's been done and we're in the process of finalizing
```

File: .changelog/v0.20.0/summary.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+this new architecture, which will form the basis for future work towards
+building a Tendermint full node/validator in Rust. More on this in future
```
Comment (xla) [inline .changelog/v0.20.0/summary.md:15]:
```suggestion
building more Tendermint nodes in Rust. More on this in future
```

**Review by xla — COMMENTED** [review]
Nice, just some smaller corrections, suggestions for the summary.

File: .changelog/v0.20.0/summary.md (hunk 2 above)
Comment (thanethomson) [reply .changelog/v0.20.0/summary.md:13]: The "thanks" part is parenthesized and emphasized :slightly_smiling_face:

Comment (xla) [reply .changelog/v0.20.0/summary.md:13]: I was suggesting the change, because it breaks the link to the user profile.

Comment (thanethomson) [reply .changelog/v0.20.0/summary.md:13]: Ah, I see. Let's just take the exclamation mark out?

```suggestion
@xla and @melekes). A lot's been done and we're in the process of finalizing
```

Comment (xla) [reply .changelog/v0.20.0/summary.md:13]: Sounds good.

**Review by xla — APPROVED** [review]
🏎 👛 👾 🍝

**Review by tarcieri — APPROVED** [review]

**Pull request merged**